mscore/plugin/api/selection.h

```diff
@@ -0,0 +1,55 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __PLUGIN_API_SELECTION_H__
+#define __PLUGIN_API_SELECTION_H__
+
+#include "elements.h"
+#include "score.h"
+
+namespace Ms {
+namespace PluginAPI {
+
+//---------------------------------------------------------
+//   Selection
+//    Wrapper class for internal Ms::Selection
+//
```

Comment:
As I mentioned in #5224, `\since MuseScore 3.3` statement could be also added here to mark the whole class as new for this version of MuseScore.

Reply:
Done.